repo/packages/C/confluent-zookeeper/100/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "Apache Zookeeper by Confluent\n\n\tDocumentation: https://docs.mesosphere.com/service-docs/confluent-zookeeper/",
+  "framework": true,
+  "selected": true,
+  "maintainer": "partner-support@confluent.io",
+  "minDcosReleaseVersion": "1.9",
```

Comment:
Is this correct? Are we keeping the minDcosReleaseVersion as 1.9 for all the data services?

Reply:
@benclarkwood @triclambert ?